course-clusters/clusters.py

```diff
@@ -0,0 +1,116 @@
+import string
+from gensim.models.doc2vec import Doc2Vec, TaggedDocument
+from load_course_info import loadCourseDesc, getSubjects
+
+subjects_url = "https://classes.cornell.edu/api/2.0/config/subjects.json?roster=FA23"
+subjects = getSubjects(subjects_url)
+
+urls = [f"https://classes.cornell.edu/api/2.0/search/classes.json?roster=FA23&subject={sub}" for sub in subjects]
+
+def fetchDescriptions(urls):
+    descriptions = []
+    for x in urls:
+        descriptions += loadCourseDesc(x)
+    print(len(descriptions))
+    return descriptions
+
+
+course_descriptions = fetchDescriptions(urls)
```

Comment:
Can we also move this to a separate file? We can add the `getSubjects` and `fetchDescriptions` functions into a separate `scrapper.py` file.